### PR TITLE
Compatibility for requestidlecallback 0.30

### PR DESCRIPTION
**Pull request by ebi**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/aFarkas/requestIdleCallback/blob/master/index.js
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/requestidlecallback/index.d.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
-// Type definitions for requestidlecallback 0.1
+// Type definitions for requestidlecallback 0.30
```
Comment (SimonSchick) [inline types/requestidlecallback/index.d.ts]: Is actually `0.3` not `0.30`

**Review by SimonSchick — CHANGES_REQUESTED** [review]

**Review by SimonSchick — APPROVED** [review]

**Comment by hax** [comment]
I don't understand this PR. We should keep using the standard API names, not internal names of polyfill.

**Comment by ebi** [comment]
@hax Could you provide a test / example then how to import and use your definitions? Which I didn't understand the functions that the library exports are the ones I defined.

Additionally the library does not support the `options` param which is defined in your declarations.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
This PR still needs clarification between definition creator and PR author

**Comment by hax** [comment]
@ebi I created this because requestidlecallback API is not available in the ts dom lib. The original commit do not reference any polyfill lib. Even consider polyfill, I suppose it should use as polyfill (which means you never use `import api from "polyfill-lib"` but just `import "polyfill-lib"`).

> how to import and use your definitions?

This is a good question, it seems we need to fix it by remove `export` keywords and use `declare` instead. It seems I made mistake in the original commit (or TS changed behavior? I'm not sure).

**Pull request closed without merging**